scripts/verify-docs.py

```diff
@@ -183,10 +223,10 @@ def validate_metadata(metadata, branch, path, fn, examples_list):
     desired_source_url = f'https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/{branch}/{path}'
     if metadata['sourceURL'] != desired_source_url:
         raise Exception(f'{fn}: "sourceURL" should be "{desired_source_url}"')
-    if len(metadata['examplePackageURLs']) != len(examples_list):
-        raise Exception(f"{fn}: examplePackageURLs have {len(metadata['examplePackageURLs'])} examples, but the examples directory has {len(examples_list)}")
+    if len(examples_list) == 0:
+        raise Exception(f'{fn}: there must be at least one example listed in metadata.yaml file')
     for example in examples_list:
-        desired_example_pkg_url = f'https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/{branch}/examples/{fn}/{example}'
+        desired_example_pkg_url = f'https://github.com/GoogleContainerTools/kpt-functions-catalog/tree/{branch}/examples/{example}'
```

Comment:
hmmm `python`. This is a decent size script now. We should try to minimize the number of languages used in the project from ease of maintenance perspective.

Reply:
Ack